.github/workflows/ci.yml

```diff
@@ -270,6 +270,39 @@ jobs:
 
           _make PLATFORM=virt
 
+  QEMUv7_check:
+    name: make check (QEMUv7)
+    runs-on: ubuntu-latest
+    container: jforissier/optee_os_ci:qemu_check
+    steps:
+      - name: Remove /__t/*
+        run: rm -rf /__t/*
+      - name: Restore build cache
+        uses: actions/cache@v3
+        with:
+          path: /github/home/.cache/ccache
+          key: qemuv7_check-cache-${{ github.sha }}
+          restore-keys: |
+            qemuv7_check-cache-
+      - name: Checkout
+        uses: actions/checkout@v4
+      - shell: bash
+        run: |
+          # make check task
+          set -e -v
+          export LC_ALL=C
+          export BR2_CCACHE_DIR=/github/home/.cache/ccache
+          export CFG_TEE_CORE_LOG_LEVEL=0
+          WD=$(pwd)
+          cd ..
+          TOP=$(pwd)/optee
+          /root/get_optee.sh default ${TOP}
+          mv ${TOP}/optee_os ${TOP}/optee_os_old
+          ln -s ${WD} ${TOP}/optee_os
```

Comment:
What and why are we doing what is done here?

Reply:
The first line clones the dev environment forest (repo init / repo sync) for platform "default", i.e., `default.xml` in https://github.com/OP-TEE/manifest.
The second and third line replace the `optee_os` project folder with a symlink to the current checkout (branch or PR for which the CI has been triggered).

Reply:
Aha, it was the second thing that wasn't clear to me. So `${WD}` can be seen as the "current PR". Thanks for the explanation.